test-resources/api-tests/events.test.ts

```diff
@@ -0,0 +1,75 @@
+import { randomUUID } from "crypto";
+import "dotenv/config";
+import {
+  createSession,
+  EVENTS_API_INSTANCE,
+  getActiveSessionId,
+  pollForEvents,
+} from "./utils/apiTestHelpers";
+
+const ONE_SECOND = 1000;
+jest.setTimeout(9 * 5 * ONE_SECOND);
```

Comment:
nit/suggestion: 45 * ONE_SECOND?
```suggestion
jest.setTimeout(45 * ONE_SECOND);
```

Reply:
updated